components/org.wso2.carbon.stream.processor.statistics/src/main/java/org/wso2/carbon/stream/processor/statistics/bean/WorkerStatistics.java

```diff
@@ -27,7 +27,8 @@
     private String runningStatus;
     private boolean isStatsEnabled;
     private String clusterID;
-    private String lastSync;
+    private String lastSync ;
```

Comment:
extra space

Reply:
remove it

Reply:
Fixed in 06dcfec1cfddb682d30ed864a9f2c63ca84a85b7